observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc

```diff
@@ -0,0 +1,37 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="observability-ui-plugins-overview"]
+= Observability UI plugins overview
+include::_attributes/common-attributes.adoc[]
+:context: observability-ui-plugins-overview
+
+toc::[]
+
+:FeatureName: The Cluster Observability Operator
+include::snippets/technology-preview.adoc[leveloffset=+2]
+
+The {coo-first} can install and manage UI plugins to enhance the observability capabilities of the {product-title} web console.
+The plugins extend the default functionality, providing new UI features for monitoring, troubleshooting, distributed tracing, and cluster logging.
+
+== Dashboards
+
+The dashboard UI plugin supports enhanced dashboards in the {product-title} web console at *Observe* -> *Dashboards*.
+You can add other Prometheus datasources from the cluster to the default dashboards, in addition to the in-cluster datasource.
+This results in a unified observability experience across different data sources.
+
+== Troubleshooting
+
+The troubleshooting panel UI plugin provides observability signal correlation, powered by the open source Korrel8r project.
+With the troubleshooting panel available under *Observe* -> *Alerting* , you can easily correlate metrics, logs, alerts, netflows, and additional observability signals and resources, across different data stores.
```

Comment:
```suggestion
You can use the troubleshooting panel available from the *Observe* -> *Alerting* page to easily correlate metrics, logs, alerts, netflows, and additional observability signals and resources across different data stores.
```

- Suggestions for active voice / minimalism and readability.
- Avoid use of "under" preposition for accessibility: https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#location-of-interface-elements (not sure if **Alerting** is a page, menu, etc, so you may need to tweak this suggestion to fit)

Reply:
done